lib/auth.js

```diff
@@ -48,6 +55,66 @@ Auth.prototype.bearer = function (bearer, sendImmediately) {
   }
 }
 
+
+Auth.prototype.negotiate = function(request, authHeader, sendImmediately) {
+  var self = this
+
+  if (kerberos === undefined) {
+    // Nothing to do if we don't have Kerberos
+    // or a current credential cache
+    return
+  }
+
+  self.hasAuth = true
+
+  if (!sendImmediately) {
```

Comment:
That makes sense. I'm using `localhost` just for the test, because the test server is listening on `localhost:6767`. The tests are actually working on my end, so you should be able to run them if you have everything setup. Of course I'm open for suggestions and improvements :+1: